src/macosx/miniAudicleController.mm

```diff
@@ -421,6 +426,13 @@ - (void)setLastWindowTopLeftCorner:(NSPoint)p
     last_window_tlc = p;
 }
 
+// activate console monitor | 1.5.0.4 (ge) this seems sus?
+- (void)activateConsoleMonitor:(id)sender
+{
+    // make appear and give focus
+    [console_monitor activateMonitor];
```

Comment:
seems fine?

Reply:
sounds good -- just wasn't sure if making a new method in miniAudicleController was the way to access `console_monitor` to activate it